packages/app/.prettierignore

```diff
@@ -1 +1,2 @@
-CHANGELOG.md
\ No newline at end of file
+CHANGELOG.md
+playwright/.auth/user.json
```

Comment:
For the `CHANGELOG.md`:

Because this file is generated by release-please and Prettier formats the markdown differently. Every time release-please the file it gets re-formatted by release-please. Due to the fact that we don't touch this file anyways, I thought we can ignore it.

For the playwright file: [this is supposed to be ignored](https://playwright.dev/docs/auth#core-concepts), I added it to the `.gitignore`. Thanks!

Reply:
Ah okay, that makes sense. Thank you!